pkg/query/config.go

```diff
@@ -39,3 +44,29 @@ func LoadConfigs(confYAML []byte) ([]Config, error) {
 	}
 	return queryCfg, nil
 }
+
+// BuildQueryConfig returns a query client configuration from a static address.
+func BuildQueryConfig(queryAddrs []string) ([]Config, error) {
+	configs := make([]Config, 0, len(queryAddrs))
+	for _, addr := range queryAddrs {
+		if addr == "" {
+			return nil, errors.New("static querier address cannot be empty")
```

Comment:
consider adding more context here, if we 2nd queryAddr is empty, it would be helpful to tell that 2nd adress is wrong